packages/rest/src/spec-enhancers/consolidate.spec-enhancer.ts

```diff
@@ -152,22 +177,22 @@ export class ConsolidationEnhancer implements OASEnhancer {
     return schema;
   }
 
-  private patchRef(
+  private static patchRef(
```

Comment:
```suggestion
  private static _patchRef(
```

Reply:
Why changing the name? Do we have a convention to use `_` for private declaration names?

Reply:
Ctrl + Shift + F on vscode shows that this naming convention is being used across several packages:

https://github.com/loopbackio/loopback-next/blob/1e3f628f1bc90ffccdb39d9d8bd4c37834fd1ff3/packages/context/src/binding.ts#L360-L377

https://github.com/loopbackio/loopback-next/blob/1e3f628f1bc90ffccdb39d9d8bd4c37834fd1ff3/packages/core/src/application.ts#L72-L81

https://github.com/loopbackio/loopback-next/blob/1e3f628f1bc90ffccdb39d9d8bd4c37834fd1ff3/packages/http-server/src/http-server.ts#L89-L94